packages/squiggle-lang/src/rescript/Distributions/SymbolicDist/SymbolicDist.res

```diff
@@ -50,18 +85,18 @@ module Normal = {
 
   let operateFloatFirst = (operation: Operation.Algebraic.t, n1: float, n2: t) =>
     switch operation {
-    | #Add => Some(#Normal({mean: n1 +. n2.mean, stdev: n2.stdev}))
-    | #Subtract => Some(#Normal({mean: n1 -. n2.mean, stdev: n2.stdev}))
-    | #Multiply => Some(#Normal({mean: n1 *. n2.mean, stdev: Js.Math.abs_float(n1) *. n2.stdev}))
+    | #Add => Some(make(~mean=n1 +. n2.mean, ~stdev=n2.stdev))
```

Comment:
`option<result<'a, 'b>>` smells off to me. I know we have it in places already, but I'm not thrilled about it.